.gitignore

```diff
@@ -1,9 +1,4 @@
-/vendor
-composer.phar
-composer.lock
-.DS_Store
-Thumbs.db
+/.phpunit.result.cache
```

Comment:
IMHO this is not good and the previous line is "better", because the current directory may be different depending on where and how you run phpunit exactly, that's the version without the leading `/` makes more sense. IMHO.